README.md

```diff
@@ -2,46 +2,134 @@
 
 source code = website
 
-> Build static websites for software projects with zero (currently a fair amount of) configuration.
+> Build a website for your software projects with one command (and zero configuration).
 
-![Logo](https://d30y9cdsu7xlg0.cloudfront.net/png/43478-200.png)
+```
+landr deploy
+```
+
+## How it works
+
+When you run landr on your local repository, it gathers info by leveraging standard conventions.
+It'll first look for a git remote from `github.com` and retrieve some basic information about your project from the github api (`contribution stats`, `releases`), it will then parse standard files like `README.md`, `CHANGELOG.md`, `CONTRIBUTING.md` & `/docs`, it'll then use the data to build out beautiful pages for your website.
+
+This allows the maintenance of your website to be a side effect of keeping your software project inline with standard github conventions.
 
-## Installation
+## Quick start guide
 
+Install:
 ```
-yarn run bootstrap
+npm i -g landr
 ```
 
+From the root of your local git repo run:
 ```
-yarn start
+landr serve
 ```
+Visit `http://localhost:8080`.
 
-Visit `localhost:8000`.
+Deploy to github pages:
+```
+landr deploy
+```
 
-## Features
+## Site composition
+
+We'll currently build these pages based on the info we can infer from a repo.
+
+- /
+  - Jumbotron (title + description of project)
+  - Installation / Getting started
+    - instructions pulled from `README.md`
+  - Features
+    - list of features
+  - Changelog (Last entry, with link to view more)
+    - link to `/contribute`
+  - Contribute
+    - (Top 5 contributors / order by commits )
+    - Issue explorer (default tag is `help wanted`)
+    - link to `/contribute`
+- /docs
+  - generates a page for each /docs/* page.
+  - Sidebar navigation
+- /changelog
+  - accordion list view of every entry from `changelog.md` or fallback to pulling from github releases
+- /contribute
+  - list of contributors
+  - stats from github api (health score/langauges etc.)
+  - instructions from `CONTRIBUTING.md`
+
+## Customizing
+
+It's great to get a full website with out touching any css/html, but you may want personalize the website by editing templates and styles, landr comes with an `eject` command to customize any piece of the website.
+
+### Styles
+```
+landr eject --styles
```

Comment:
In that case I think we should have an `eject-styles` command and other similar ones. Options should be optional, since that is what a user would expect. Let's either make this command do something if no options are passed, or create different commands.